### PR TITLE
Fixes for the Android REST API section

### PR DESCRIPTION
**Pull request by jpignata**
*Issue #, if available:* n/a

*Description of changes:* ⬆️ 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/restapi/fragments/android/getting-started/11_amplifyInit.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+? Choose the function template that you want to use: 
+    `Serverless ExpressJS function (Integration with API Gateway)`
+? Do you want to access other resources created in this project from your Lambda function? 
+    `No`
+? Do you want to invoke this function on a recurring schedule? 
+    `No`
+? Do you want to edit the local lambda function now?
+    `No`
+? Restrict API access 
+    `No`
```
Comment (lawmicha) [inline docs/lib/restapi/fragments/android/getting-started/11_amplifyInit.md:32]: should we link to https://github.com/aws-amplify/amplify-android/issues/537 as a TODO comment in this file to remember to consolidate the provisioning back to using auth, or perhaps we should just disable auth all together for ios as well for an easier run-through?

File: docs/lib/restapi/fragments/android/getting-started/11_amplifyInit.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+amplify add api
+```
+
+Enter the following when prompted:
+```console
+? Please select from one of the below mentioned services: 
+    `REST`
+? Provide a friendly name for your resource to be used as a label for this category in the project: 
+    `api`
+? Provide a path (e.g., /book/{isbn}):
```
Comment (undefobj) [inline docs/lib/restapi/fragments/android/getting-started/11_amplifyInit.md:13]: Have you tested the functionality if a customer does setup a greedy path like this?

File: docs/lib/restapi/fragments/android/getting-started/30_initapi.md [hunk 3]
```diff
@@ -19,8 +18,7 @@ public class MyAmplifyApp extends Application {
         super.onCreate();
 
         try {
-            // Add these lines to add the AWSCognitoAuthPlugin and AWSApiPlugin plugins
-            Amplify.addPlugin(new AWSCognitoAuthPlugin());
+            // Add this line to add the AWSApiPlugin plugin
```
Comment (lawmicha) [inline docs/lib/restapi/fragments/android/getting-started/30_initapi.md:21]: at some point a developer will have to deal with `amplify update api` and restrict access. currently this is more in line with API graphQL docs which use API Key. but i wonder, if we should consolidate ios REST API to not use auth as well, and have a new section that shows them how to `amplify update api`, restrict access, and add auth plugin

File: docs/lib/restapi/fragments/android/getting-started/11_amplifyInit.md (hunk 2 above)
Comment (jpignata) [reply docs/lib/restapi/fragments/android/getting-started/11_amplifyInit.md:13]: Not yet, no. Added to our list to test.

File: docs/lib/restapi/fragments/android/getting-started/11_amplifyInit.md (hunk 1 above)
Comment (jpignata) [reply docs/lib/restapi/fragments/android/getting-started/11_amplifyInit.md:32]: I'm not opposed to either.

File: docs/lib/restapi/fragments/android/getting-started/30_initapi.md (hunk 3 above)
Comment (jpignata) [reply docs/lib/restapi/fragments/android/getting-started/30_initapi.md:21]: Not opposed.

**Pull request merged**